core/src/Revolution/Processors/Object/CreateProcessor.php

```diff
@@ -8,9 +8,11 @@
  * file that was distributed with this source code.
  */
 
-namespace MODX\Revolution;
+namespace MODX\Revolution\Processors\Object;
```

Comment:
I would refrain from using [reserved words](https://www.php.net/manual/en/reserved.other-reserved-words.php)(object; as of PHP 7.2) in namespaces.
> Reserved words cannot be used to name a class, interface or trait, and they are also prohibited from being used in namespaces.

Reply:
That... does seem like a problem. I'm surprised phpstorm isn't flagging that.. 

It's getting late and my brain is fairly fried, but I can think of 2 alternatives.

1. Rename the namespace to `xPDOObject` (e.g. `\MODX\Revolution\Processors\xPDOObject\CreateProcessor`), so you still benefit from the clean `class MyProcessor extends CreateProcessor`. 
2. Move up one directory/namespace and reintroduce `Object` into the classname (e.g. `\MODX\Revolution\Processors\ObjectCreateProcessor`). Still some benefit from pulling processors out of `\MODX\Revolution`, but not quite as neat in its own directory. 

Good catch!

Reply:
> That... does seem like a problem. I'm surprised phpstorm isn't flagging that..

It's possible, but prohibited, to use them.

> It's getting late and my brain is fairly fried, but I can think of 2 alternatives.
> 
> 1. Rename the namespace to `xPDOObject` 
> 2. Move up one directory/namespace and reintroduce `Object` into the classname 

I have a third alternative: rename `Object` to `Model` (e.g. `\MODX\Revolution\Processors\Model\CreateProcessor`) 

> Good catch!

Thanks! 😄

Reply:
Updated to use `Model` - that's a pretty good alternative.